shelley/chain-and-ledger/shelley-spec-ledger-test/shelley-spec-ledger-test.cabal

```diff
@@ -170,6 +172,7 @@ test-suite shelley-spec-ledger-test
       cardano-crypto-test,
       cardano-crypto-wrapper,
       cardano-crypto,
+      cardano-ledger-shelley-ma,
```

Comment:
I guess ideally we would prefer to have the tests in a shelley-ma package. But this is okay for now. Maybe make a ticket to cover the work of setting up the `shelley-ma-test` package and moving things there?

Reply:
I agree. Eventually I want to add some benchmarks as well. And they should also go in the shelley-ma  package.